### PR TITLE
Make performance test save results in csv file

### PR DESCRIPTION
**Pull request by Mykhailenko**
This PR implements "Log all performance data from Performance test to logs/Scheduler-performance-tests.csv", "Increase scope of performance testing from running job recovery to job recovery" and  "add bigger arguments for performance tests by increasing timeout" from [here](https://trello.com/c/SvGlMvbB/2874-aggregate-results-in-csv-file).

### CODE REVIEW

File: scheduler/scheduler-server/src/test/java/performancetests/recovery/NodeRecoveryTest.java [hunk 1]
```diff
@@ -112,14 +116,33 @@ public void startKillStartRM() throws Exception {
         assertEquals(nodesNumber, resourceManager.getState().getAllNodes().size());
     }
 
+    @Ignore
     @Test
     public void test() {
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/test/java/performancetests/recovery/NodeRecoveryTest.java]: MAJOR Fix or remove this skipped unit test [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1607)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SchedulerDBManager.java#L1733](https://github.com/ow2-proactive/scheduling/blob/789381c5264f0478bd4418d6721cef4493841330/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java#L1733): Remove this unused private "taskId" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [SchedulerDBManager.java#L1744](https://github.com/ow2-proactive/scheduling/blob/789381c5264f0478bd4418d6721cef4493841330/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java#L1744): Remove this unused private "jobId" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [SchedulingService.java#L626](https://github.com/ow2-proactive/scheduling/blob/789381c5264f0478bd4418d6721cef4493841330/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L626): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L658](https://github.com/ow2-proactive/scheduling/blob/789381c5264f0478bd4418d6721cef4493841330/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L658): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L690](https://github.com/ow2-proactive/scheduling/blob/789381c5264f0478bd4418d6721cef4493841330/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L690): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L720](https://github.com/ow2-proactive/scheduling/blob/789381c5264f0478bd4418d6721cef4493841330/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L720): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L748](https://github.com/ow2-proactive/scheduling/blob/789381c5264f0478bd4418d6721cef4493841330/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L748): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Mykhailenko** [comment]
retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SchedulerDBManager.java#L1733](https://github.com/ow2-proactive/scheduling/blob/85fb847cc76d5d05d288cf777b7e30903c51cea8/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java#L1733): Remove this unused private "taskId" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [SchedulerDBManager.java#L1744](https://github.com/ow2-proactive/scheduling/blob/85fb847cc76d5d05d288cf777b7e30903c51cea8/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java#L1744): Remove this unused private "jobId" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [SchedulingService.java#L626](https://github.com/ow2-proactive/scheduling/blob/85fb847cc76d5d05d288cf777b7e30903c51cea8/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L626): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L658](https://github.com/ow2-proactive/scheduling/blob/85fb847cc76d5d05d288cf777b7e30903c51cea8/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L658): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L690](https://github.com/ow2-proactive/scheduling/blob/85fb847cc76d5d05d288cf777b7e30903c51cea8/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L690): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L720](https://github.com/ow2-proactive/scheduling/blob/85fb847cc76d5d05d288cf777b7e30903c51cea8/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L720): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L748](https://github.com/ow2-proactive/scheduling/blob/85fb847cc76d5d05d288cf777b7e30903c51cea8/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L748): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SchedulerDBManager.java#L1733](https://github.com/ow2-proactive/scheduling/blob/da8c69e275ad775f327d8d92fe29a493f0e29b55/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java#L1733): Remove this unused private "taskId" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [SchedulerDBManager.java#L1744](https://github.com/ow2-proactive/scheduling/blob/da8c69e275ad775f327d8d92fe29a493f0e29b55/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java#L1744): Remove this unused private "jobId" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [SchedulingService.java#L626](https://github.com/ow2-proactive/scheduling/blob/da8c69e275ad775f327d8d92fe29a493f0e29b55/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L626): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L658](https://github.com/ow2-proactive/scheduling/blob/da8c69e275ad775f327d8d92fe29a493f0e29b55/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L658): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L690](https://github.com/ow2-proactive/scheduling/blob/da8c69e275ad775f327d8d92fe29a493f0e29b55/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L690): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L720](https://github.com/ow2-proactive/scheduling/blob/da8c69e275ad775f327d8d92fe29a493f0e29b55/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L720): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulingService.java#L748](https://github.com/ow2-proactive/scheduling/blob/da8c69e275ad775f327d8d92fe29a493f0e29b55/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java#L748): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request closed without merging**